snippets/language-svg.cson

```diff
@@ -1,101 +1,352 @@
+  'group':
+    'prefix': 'g'
+    'body': '<g $1>$2</g>'
+    'description': 'Container for grouping together related graphics elements.'
+    'descriptionMoreURL': 'http://www.w3.org/TR/SVG/struct.html#Groups'
+  'hyperlink':
+    'prefix': 'a'
+    'body': '  <a xlink:href="$1" target="${2:_blank}">$3</a>'
+    'description': 'The SVG Anchor Element (<a>) defines a hyperlink'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/a'
+  'image':
+    'prefix': 'image'
+    'body': '<image xlink:href="$1" x="${2:0}" y="${3:0}" height="${4:100}" width="${5:100}" /> '
+    'description': 'The SVG Image Element (<image>) allows a raster image into be included in an SVG document.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image'
+  'line':
+    'prefix': 'l'
+    'body': ' <line x1="$1" y1="$2" x2="$3" y2="$4" stroke="$5" stroke-width="$6"/>'
+    'description': 'The line element is an SVG basic shape, used to create a line connecting two points.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/de/docs/Web/SVG/Element/line'
   'linearGradient':
     'prefix': 'lg'
     'body': '<linearGradient id="$1" x1="${2:0}" x2="${3:0}" y1="${4:0}" y2="${5:1}">\n  <stop offset="${6:0}" stop-color="$7" />\n  <stop offset="${8:1}" stop-color="$9" />\n</linearGradient>'
+    'description': ''
+    'descriptionMoreURL': ''
+  'mask':
+    'prefix': 'mask'
+    'body': '<mask id="$1" maskUnits="${2:userSpaceOnUse}" x="${3:0}" y="${4:0}" width="${5:200}" height="${6:80}">$7</mask>'
+    'description': 'In SVG, you can specify that any other graphics object or <g> element can be used as an alpha mask for compositing the current object into the background. A mask is defined with the mask element. A mask is used/referenced using the mask property.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Element/mask'
   'path':
     'prefix': 'p'
     'body': '<path fill="$1" stroke="$2" stroke-width="1" \n      d="$3" />\n'
+    'description': 'The path element is the generic element to define a shape. All the basic shapes can be created with a path element.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/de/docs/Web/SVG/Element/path'
   'pattern':
     'prefix': 'pa'
     'body': '<pattern id="$1" width="$2" height="$3">\n  $4\n</pattern>'
+    'description': 'A pattern is used to fill or stroke an object using a pre-defined graphic object which can be replicated ("tiled") at fixed intervals in x and y to cover the areas to be painted. Patterns are defined using the pattern element and then referenced by properties fill and stroke on a given graphics element to indicate that the given element shall be filled or stroked with the referenced pattern.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/de/docs/Web/SVG/Element/pattern'
+  'polygon':
+    'prefix': 'polygon'
+    'body': '<polygon points="$1,$2 $3,$4 $5,$6 $7,$8 $9,$10 $11,$12"/>'
```

Comment:
wouldn't it be enough just to have a single $ here? or perhaps two for a full coordinate.